docs/extensions/FrameworkElementEx.md

```diff
@@ -0,0 +1,42 @@
+---
+title: FrameworkElement extentions
+author: ST-Apps
+ms.date: 01/05/2018
+description: FrameworkElementEx provides a simple way to bind to actual size for any FrameworkElement
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, FrameworkElement, extentions
+---
+
+# FrameworkElement extentions
+
+FrameworkElementEx provides a simple way to bind to actual size for any FrameworkElement
+
+## EnableActualSizeBinding
+
+The EnableActualSizeBinding property allows to enable/disable the binding for ActualHeight and ActualWidth
```

Comment:
The EnableActualSizeBinding property allows *you* to enable/disable the binding for *the* ActualHeight and ActualWidth *extensions.*

Suggested changes in italics.